### PR TITLE
Two Factor Authentication

### PR DESCRIPTION
**Pull request by ultramegatom**
Two Factor Authentication

### CODE REVIEW

File: doc/Development_Documentation/18_Tools_and_Features/40_Two_Factor_Authentication.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Pimcore has an integrated two factor authentication using the Google Authenticator ([Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2), [iOS](https://itunes.apple.com/at/app/google-authenticator/id388497605))
+
+## Setup
+You can enable it in the profile settings menu
+ ![Settings](../img/Icon_settings.png)
+**(Settings -> My Profile)**
+
+After enabling it a secret will be generated and you can setup your Google Authenticator App.
+**Please be aware if you don't setup the App properly wou will loose access to your account!**
```
Comment (ThomasKeil) [inline doc/Development_Documentation/18_Tools_and_Features/40_Two_Factor_Authentication.md]: There's a typo, wou -> you

Comment (ultramegatom) [reply doc/Development_Documentation/18_Tools_and_Features/40_Two_Factor_Authentication.md]: thx

File: pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/translations/en.json [hunk 2]
```diff
@@ -1893,5 +1893,16 @@
   "keybinding_roles": "Roles",
   "keybinding_clearAllCaches": "Clear all Caches",
   "keybinding_clearDataCache": "Clear Data Cache",
-  "please_dont_forget_to_reload_pimcore": "<b>Plesae don't forget to reload pimcore after your modification!</b>"
+  "please_dont_forget_to_reload_pimcore": "<b>Plesae don't forget to reload pimcore after your modification!</b>",
```
Comment (ThomasKeil) [inline pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/translations/en.json]: Typo: Plesae -> Please

File: pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/translations/en.json [hunk 3]
```diff
@@ -1893,5 +1893,16 @@
...
+  "2fa_settings": "Two Factor Authentication",
+  "2fa_enabled": "Two Factor Authentication Enabled",
+  "2fa_secret": "Secret",
+  "2fa_code": "Code",
+  "2fa_renew_button": "Save Two Factor Settings",
+  "2fa_alert_title": "Two Factor Authentication",
+  "2fa_alert_text": "Please scan the QR-Code and add it to your Google Authenticator. <br><br> You will need to reload Pimcore and and enter the code shown within the App afterwards!",
```
[the remark this answers is not in the document]
Comment (ultramegatom) [reply pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/translations/en.json]: thx

**Pull request merged**